assets/templates/signup.html

```diff
@@ -37,42 +37,46 @@
         <small class="content-signin">¿Ya tienes una cuenta? <a href="#" class="link-green" target="_blank">INGRESA</a></small>
       </header><!-- header -->
       <h5>Regístrate</h5>
-      <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p><!-- Title -->
+      <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p><!-- title and subtitle -->
       <form class="signup-form">
         <fieldset>
           <legend>Creación de cuenta</legend>
-          <ul>
-            <li class="holder-small">
-              <label for="name">Nombre:</label>
-              <input type="name" name="lastname" placeholder="Nombre">
+          <ul class="holder-group">
+            <li class="holder-size1of1">
+              <ul class="holder-group holder-col">
+                <li class="holder-size1of2">
+                  <label for="name">Nombre:</label>
+                  <input type="name" name="lastname" placeholder="Nombre">
+                </li>
+                <li class="holder-size1of2">
+                  <label for="lastname">Apellido:</label>
+                  <input type="lastname" name="name" placeholder="Apellido">
+                </li>
+              </ul>
             </li>
-            <li class="holder-small">
-              <label for="lastname">Apellido:</label>
-              <input type="lastname" name="name" placeholder="Apellido">
-            </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="phone">Teléfono:</label>
               <input type="tel" name="phone" placeholder="9 11 2345-6789">
             </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="email">Email:</label>
               <input type="email" name="email" placeholder="Tu Email será tu nombre de usuario">
             </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="password">Contraseña:</label>
               <input type="password" name="password" placeholder="Escribe tu clave secreta">
```

Comment:
Inputs should have the `id` attribute with a value that matches the `name` attribute of it's label for the label to be clickeable (so it can focus on the input). Same goes with all of the inputs.

Reply:
Ok!